pages/course/[courseId]/index.tsx

```diff
@@ -27,22 +35,24 @@ export default function Details() {
   const course = coursesById[0];
 
   //TODO: change later, when implementing the services into this side
```

Comment:
This ran be removed now, right?

Reply:
You mean the TODO? Not really, because it has still hardcoded numbers for the skill level, because we still do not have the skill level application.